flixel/text/FlxText.hx

```diff
@@ -33,6 +33,11 @@ using StringTools;
 class FlxText extends FlxSprite
 {
 	/**
+	 * 2px gutter on both top and bottom
+	 */
+	private static var VERTICAL_GUTTER_AMOUNT : Int = 4;
```

Comment:
Several things:

- this var should be `inline`
- there shouldn't be any spaces surrounding the `:` (see the [code styleguide](http://haxeflixel.com/documentation/code-style/) (Colon Spacing))
- I don't think the `_AMOUNT` really adds any information